tests/block/test_get_blocks.py

```diff
@@ -0,0 +1,328 @@
+"""
+Provide tests for command line interface's get a list of blocks command.
+"""
+import json
+
+import pytest
+from click.testing import CliRunner
+
+from cli.constants import (
+    FAILED_EXIT_FROM_COMMAND_CODE,
+    NODE_IP_ADDRESS_FOR_TESTING,
+    PASSED_EXIT_FROM_COMMAND_CODE,
+)
+from cli.entrypoint import cli
+from cli.utils import dict_to_pretty_json
+
+
+def test_get_blocks():
+    """
+    Case: get a list of blocks.
+    Expect: blocks are returned.
+    """
+    runner = CliRunner()
+    result = runner.invoke(cli, [
+        'block',
+        'get-list',
+        '--node-url',
+        NODE_IP_ADDRESS_FOR_TESTING,
+    ])
+
+    list_of_blocks = json.loads(result.output).get('result')
+
+    assert PASSED_EXIT_FROM_COMMAND_CODE == result.exit_code
+    assert isinstance(list_of_blocks, list)
+
+    for block in list_of_blocks:
+        assert isinstance(block.get('batches'), list)
+
+
+def test_get_blocks_with_ids():
+    """
+    Case: get a list of blocks by identifiers.
+    Expect: blocks with header signatures which matches specified identifiers are returned.
+    """
+    blocks_ids = 'fe56a16dab009cc96e7125c647b6c71eb1063818cf8dece283b125423ecb184f' \
+                 '7f1e61802bf66382da904698413f80831031f8a1b29150260c3fa4db537fdf4c, ' \
+                 '56100bf24eed12d2f72fe3c3ccf75fe2f53d87c224d9dda6fb98a1411070b06a' \
+                 '40fcf97fccc61cb9c88442953af6ae50344ad7773f1becc6bae108443c18c551'
+
+    runner = CliRunner()
+    result = runner.invoke(cli, [
+        'block',
+        'get-list',
+        '--ids',
+        blocks_ids,
+        '--node-url',
+        NODE_IP_ADDRESS_FOR_TESTING,
+    ])
+
+    list_of_blocks = json.loads(result.output).get('result')
+
+    assert PASSED_EXIT_FROM_COMMAND_CODE == result.exit_code
+
+    for block in list_of_blocks:
+        assert block.get('header_signature') in blocks_ids
+
+
+def test_get_blocks_invalid_ids():
```

Comment:
Create a test for non-existing identifiers and limit.

Reply:
@anastasiia-bilova, done.